resources/nodes/InertialUnit.wrl

```diff
@@ -21,7 +21,7 @@ InertialUnit {
   field     SFFloat    radarCrossSection   0.0              # radar cross section of this solid
   field     MFColor    recognitionColors   []               # colors returned for this Solid by Cameras with a Recognition node
   #fields specific to the InertialUnit node:
-  field     MFVec3f    lookupTable     []          # empty: no interpolation
```

Comment:
Usually, the field is actually kept in the '*.wrl' file (at the end) with the 'deprecatedField' keyword instead of 'field' so that a deprecation warning is automatically generated if needed. See for example: https://github.com/cyberbotics/webots/blob/master/resources/nodes/Camera.wrl#L46